<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test10Words</class>
 <widget class="QDialog" name="Test10Words">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тест</string>
  </property>
  <widget class="QLabel" name="word5">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>159</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>164</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5)</string>
   </property>
  </widget>
  <widget class="QLabel" name="word7">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>39</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 7</string>
   </property>
  </widget>
  <widget class="QLabel" name="word3">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>80</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>4</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>204</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>124</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4)</string>
   </property>
  </widget>
  <widget class="QLabel" name="word8">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>75</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>4</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>124</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9)</string>
   </property>
  </widget>
  <widget class="QLabel" name="word4">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>119</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="word9">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>119</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>44</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7)</string>
   </property>
  </widget>
  <widget class="QLabel" name="word1">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>0</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_e">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>164</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10)</string>
   </property>
  </widget>
  <widget class="QLabel" name="word6">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>0</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 6</string>
   </property>
  </widget>
  <widget class="QLabel" name="word_10">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>159</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>84</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>44</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>204</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QLabel" name="word2">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>39</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слово 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>84</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
